frontend/src/components/ContactUs.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import { useState, useEffect} from 'react'
+import { TextField, Grid, makeStyles, Paper, Button, Typography } from '@material-ui/core'
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles'
+
+
+const useStyles = makeStyles({
+    root:{
+        '& .MuiFormControl-root':{
+            width: '80%',
+            margin: 8,
+        },
+      
+    },
+    pageSideBar:{
+        margin: 8,
+        padding: 50,
+        backgroundColor: "rgb(238,235,228)",
+        
+    },
+   
+    pageContent:{
+        margin: 8,
+        padding: 110,
+        backgroundColor: "rgb(234,232,224)",
+        
+    },
+    submitButton:{
+        margin: 1,
+        
+    },
+    cancelButton:{
+        margin: 1,
+
+    },
+    sideBar:{
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+        '@media (min-width:1100px)':{
+            fontSize: '1.5rem',
+        },
+
+    },
+
+})
+
+const contactValue = {
+    name: '',
+    email: '',
+    message: '',
+
+}
+
+export default function ContactUs() {
+    const [values, setValues] = useState(contactValue);
+    const classes = useStyles();
+
+    const getInput= e =>{
+        const {name, value } = e.target
+        setValues({
+            ... values,
+            [name]: value
+        })
+    }
+
+    return (
+        <>
+       
+        <Grid container>
+            <Paper className={classes.pageSideBar}>
```

Comment:
Any reason you didn't just apply the class to the Grid? What does Paper add? I've never used it before.

Reply:
Paper just applies a background that resembles a flat paper.  So I just put the grid inside the paper.  I can try apply the class to the Grid.